cogs/meta.py

```diff
@@ -23,7 +56,20 @@ async def check(self, ctx):
         except Exception as e:
             await ctx.send('Seems like I\'m getting cloud flared, rip. ' +
                            str(e))
-
+    @commands.command()
+    async def info(self, ctx):
```

Comment:
As much as I appreciate the links and docs, it's not necessary.

Reply:
definitely not essential, i just think it's nice to have for new ppl